### PR TITLE
Put Whitenoise back in settings.py

### PR DESCRIPTION
**Pull request by jfmatth**
made a simple change to settings.py to put Whitenoise back

### CODE REVIEW

**Comment by bparees** [comment]
@rhcarvalho ptal

**Comment by rhcarvalho** [comment]
Why was it removed? PR link?

**Comment by jfmatth** [comment]
I think when white noise was upgraded it was accidentally removed

**Comment by jfmatth** [comment]
Here is the original one with 'removed' the whitenoise.  When moving from 2.x to 3.x, it was a design changed where whitenoise was used from, in 3.x, it's used as a middleware now, before I don't recall

https://github.com/openshift/django-ex/pull/40/commits

File: project/settings.py [hunk 1]
```diff
@@ -55,6 +55,8 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```
Comment (rhcarvalho) [inline project/settings.py]: The [docs](http://whitenoise.evans.io/en/stable/django.html#enable-whitenoise) say to place it after `django.middleware.security.SecurityMiddleware`, so this looks good :+1:

File: project/settings.py [hunk 2]
```diff
@@ -55,6 +55,8 @@
...
+
```
Comment (rhcarvalho) [inline project/settings.py]: @jfmatth could you get rid of this extraneous empty line please?

**Comment by rhcarvalho** [comment]
Looks like #40 bumped the version without updating the code to the new usage instructions.

@jfmatth could you please remove `DjangoWhiteNoise` from `project/wsgi.py`?

We shouldn't have it setup twice both as a Django middleware and a WSGI middleware.

**Comment by rhcarvalho** [comment]
http://whitenoise.evans.io/en/stable/django.html#add-compression-and-caching-support would be nice to have, but not required here.
I created #52 to track that.

**Comment by rhcarvalho** [comment]
I see this is related to #39.

**Comment by jfmatth** [comment]
Removed extra line

**Comment by rhcarvalho** [comment]
Need to change https://github.com/openshift/django-ex/blob/master/project/wsgi.py#L16-L18 to remove the WhiteNoise middleware, squash commits, then LGTM :)

**Comment by jfmatth** [comment]
I'll take care of this tonight

**Comment by rhcarvalho** [comment]
@jfmatth thank you!

**Comment by jfmatth** [comment]
I was unable to test on the dev preview instance since it's having issues :(

**Comment by jfmatth** [comment]
Got my preview to work, tested the compression and the new whitenoise
branches

**Pull request merged**